pkg/operator/ceph/object/health.go

```diff
@@ -88,7 +88,7 @@ func (c *bucketChecker) checkObjectStore(stopCh chan struct{}) {
 			return
 
 		case <-time.After(c.interval):
-			logger.Debug("checking rgw health of object store %q", c.namespacedName.Name)
+			logger.Debugf("checking rgw health of object store %q", c.namespacedName.Name)
```

Comment:
You have a similar commit in #5837, could you combine them to a single commit either in this PR or the other one?